### ISSUE TITLE
Please update the unfair comparison with ShiftAddNet

### ISSUE THREAD

**Issue by ranery** [huawei-noah/AdderNet#43]
Hi, 

We recently noticed that you update the comparison with ShiftAddNet in your readme.

However, the chosen accuracy of ShiftAddNet (i.e., 85.10%) is **wrongly** from our adaption experiments, for which we only use half the CIFAR-10 dataset to pre-train. Our accuracy (ResNet-20 on CIFAR-10) should be 89.32%, which is also trained with a different setting and different implementation (e.g., only 160 epochs).

Could you help to update the comparison results?

Best regards,
Haoran You

**Comment by HantingChen** [comment]
Sorry for the mistake. The results have been updated.

**Comment by ranery** [comment]
Thanks!

**Comment by brisker** [comment]
@ranery
Hi, so if shiftaddnet follow the same training setting as the original addernet paper, will you also get 91.8 performance for resnet20-cifar10 ??
